crates/orderbook/openapi.yml

```diff
@@ -1105,10 +1109,14 @@ components:
         with.
       type: object
       properties:
+        auctionId:
+          type: number
```

Comment:
I believe this should be `integer` and to so that it has the same type as the id of the `Auction` type.